docs/users.rst

```diff
@@ -759,3 +759,13 @@ releases.
 
 
 .. _`config.json`: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/config.json
+
+With enabled  :ref:`OMPS integration <omps-integration>` operator manifests are
+after successful build uploaded into configured application registry and
+namespace.
+
+Details how operator manifest can be accessed from application registry are
+stored in koji build, in section ``build.extra.operator_manifests.appregistry``.
+
+Manifests from scratch, isolated and re-builds are not pushed into application
```

Comment:
How about: Manifests will not be pushed to the application registry for scratch builds, isolated builds, or re-builds to prevent unwanted changes ?